code/src/utest/pcgen/cdom/facet/fact/WeightFacetTest.java

```diff
@@ -40,41 +40,33 @@ public void setUp() throws Exception
 	@Test
 	public void testWeightUnsetZero()
 	{
-		assertEquals(0, facet.getWeight(id));
+		assertEquals(0, (int)facet.get(id));
 	}
 
 	@Test
 	public void testWeightSetZeroValid()
 	{
-		facet.setWeight(id, 0);
-		assertEquals(0, facet.getWeight(id));
+		facet.set(id, 0);
+		assertEquals(0, (int)facet.get(id));
 	}
 
 	@Test
 	public void testWeightSetNegative()
 	{
-		facet.setWeight(id, -250);
+		facet.set(id, -250);;
 		/*
 		 * TODO Some form of error here?
 		 */
 	}
 
-	@Test
-	public void testRemoveWeight()
-	{
-		facet.setWeight(id, 25);
-		assertEquals(25, facet.getWeight(id));
-		facet.removeWeight(id);
-		assertEquals(0, facet.getWeight(id));
-	}
-
 	@Override
 	protected AbstractItemFacet<CharID, Integer> getFacet()
 	{
 		return facet;
 	}
 
-	private int n = 1;
+	// random value > 0
+	private int n = 123242;
```

Comment:
the real reason for this is that in AgeFacet it was originally 0 which caused confusion with the newly implemented valueWhenNull().
the secondary reason is that test values should typically be somewhat arbitrary.